base/markdown/render/plain.jl

```diff
@@ -38,6 +38,14 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, q::BlockQuote)
+    s = sprint(buf -> plain(buf, q.content))
```

Comment:
I guess there's no way around this (reading the whole thing), but this feels inefficient/like it could be done in a single pass...